godtools/ViewControllers/Platform/LanguagesTableViewController+TableDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
I like this way of setting the extension, like a category in the past, maybe we should think using this pattern for other extensions